maps/RadioTower5F.asm

```diff
@@ -6,10 +6,11 @@
 	const RADIOTOWER5F_POKE_BALL
 
 RadioTower5F_MapScripts:
-	db 3 ; scene scripts
+	db 4 ; scene scripts
 	scene_script .DummyScene0 ; SCENE_DEFAULT
 	scene_script .DummyScene1 ; SCENE_RADIOTOWER5F_ROCKET_BOSS
 	scene_script .DummyScene2 ; SCENE_RADIOTOWER5F_NOTHING
+	scene_script .DummyScene3
```

Comment:
i don't love that this adds a lot of duplicate code. i would prefer refactoring the whole thing so that the rocket boss is an NPC that you can talk to. it's up to you though.

Reply:
Using scenes (triggers) is probably the easiest way I see to do this. It ensures that all NPCs maintain their vanilla behavior, which is less confusing for players. The duplication of code shouldn't really be much of a concern here (this IS assembly code after all).

Reply:
that's fine.